### PR TITLE
Cards vue I

### PR DESCRIPTION
**Pull request by chosta**
All I cards and a couple from previous letters that needed some special attention.

### CODE REVIEW

File: src/cards/base/Herbivores.ts [hunk 1]
```diff
@@ -43,16 +44,17 @@ export class Herbivores implements IProjectCard, IResourceCard {
...
           eb.greenery().startEffect.animals(1);
           eb.description('Effect: When you place a greenery tile, add an Animal to this card.');
         }).br;
-        b.animals(1).productionBox((pb) => pb.minus().plants(1).any).br;
         b.text('1 VP per 2 Animals on this card', CardRenderItemSize.TINY, true);
+        b.animals(1).productionBox((pb) => pb.minus().plants(1).any);
       }),
       description: {
-        text: 'Requires 8% oxygen. Add 1 Animal to this card. Decrease any Plant production 1 step.',
+        text: 'Requires 8% oxygen. +1 animal to this card. -1 plant production',
```
Comment (nwai90) [inline src/cards/base/Herbivores.ts:57]: Is the text change intentional? It looks a little strange

File: src/cards/base/ImportedNitrogen.ts [hunk 2]
```diff
@@ -32,4 +34,14 @@ export class ImportedNitrogen implements IProjectCard {
       game.defer(new AddResourcesToCard(player, game, ResourceType.ANIMAL, 2));
       return undefined;
     }
+    public metadata: CardMetadata = {
+      cardNumber: '163',
+      renderData: CardRenderer.builder((b) => {
+        b.tr(1).br;
+        b.plants(4).br;
+        b.microbes(3).digit.asterix().nbsp.or().nbsp;
```
Comment (nwai90) [inline src/cards/base/ImportedNitrogen.ts:42]: We should remove the `.or()` here as it is `AND` for this card

File: src/cards/base/Herbivores.ts (hunk 1 above)
Comment (chosta) [reply src/cards/base/Herbivores.ts:57]: I'm fighting for every pixel here. It's that or no description as I don't want to make custom css for separate cards. (at least not yet)

File: src/cards/base/ImportedNitrogen.ts (hunk 2 above)
Comment (chosta) [reply src/cards/base/ImportedNitrogen.ts]: oops

File: src/cards/base/Herbivores.ts (hunk 1 above)
Comment (kberg) [reply src/cards/base/Herbivores.ts:57]: How about restoring the original text for the moment, and // TODO to indicate this is too long? I caution that omitting the "any" needs to be restored regardless.

File: src/cards/base/InventionContest.ts [hunk 3]
```diff
@@ -27,4 +29,10 @@ export class InventionContest implements IProjectCard {
         return undefined;
       });
     }
+    public metadata: CardMetadata = {
+      cardNumber: '192',
+      renderData: CardRenderer.builder((b) => {
+        b.text('look at the top 3 cards from the deck. take 1 of them into hand and discard the other two', CardRenderItemSize.SMALL, true);
```
Comment (kberg) [inline src/cards/base/InventionContest.ts:35]: These sentences aren't capitalized.

File: src/cards/base/Herbivores.ts (hunk 1 above)
Comment (kberg) [reply src/cards/base/Herbivores.ts:57]: How many cards have you rewritten from their original text?

**Review by kberg — APPROVED** [review]

File: src/cards/base/InventionContest.ts (hunk 3 above)
Comment (chosta) [reply src/cards/base/InventionContest.ts:35]: No need to as the text is capital letters anyway. But will do, for the sake of punctuation.

File: src/cards/base/Herbivores.ts (hunk 1 above)
Comment (chosta) [reply src/cards/base/Herbivores.ts:57]: `Any` needs to be there, I agree. I've changed 2 cards so far. Will figure something out. Maybe a text-overflow propery or smth. Don't want a custom `css` positioning for it but if we have to do it, I'll do it.

**Pull request merged**